cps/web.py

```diff
@@ -990,7 +991,7 @@ def get_update_status():
     status = {}
     if request.method == "GET":
         # should be automatically replaced by git with current commit hash
-        commit_id = '$Format:%H$'
+        commit_id = '66c1966b44454828b0dd431c2d07c5b71c064598'
```

Comment:
I'm new person in this person and mostly as end-user (sorry, I don't know python). Why this changed? (and line 2472 as well)

Reply:
Both lines are used by git during commit/push and replaced automatically with the current commit stamp. To make the auto update working, it should be the original text. But I can change this on merge.